frontend/src/routes/AccessControlManagement/CreateAccessControlManagementPage.tsx

```diff
@@ -42,15 +42,7 @@ const CreateAccessControlManagementPage = () => {
           </AcmPage>
         )
       default:
-        return (
-          <AccessControlManagementForm
-            isCreatable={true}
-            namespaces={projects}
-            isEditing={false}
-            isViewing={false}
-            credentialsType={Provider.ansible}
-          />
-        )
+        return <AccessControlManagementForm isCreatable={true} isEditing={false} isViewing={false} />
```

Comment:
very minor tech suggestion
```suggestion
        return <AccessControlManagementForm isEditing={false} isViewing={false} />
```